README.md

```diff
@@ -86,6 +86,18 @@ cargo run -- \
   --bucket foobar
 ```
 
+If you just need to use the local disk as the backend, use the following bash.
+
+```bash
+# Change this to the output of `openssl rand -hex 32`.
+export RUDOLFS_KEY=282013cfb408da0a27079f515cd76f8793f6565a86e90b97ba36787fe14c6966
+
+
+cargo run -- \
+  --port 8080 \
+  local --path=/data
```

Comment:
This is a pretty short command. Can this all go on one line?

Reply:
solved.